### DISCUSSION TITLE
Additional information on SC 1.1.1 Non-text Content

### DISCUSSION THREAD

**Discussion by cmuncharaz** [w3c/wcag#2523] [Q&A] [answered]
I have a doubt regarding a fragment included in the additional information section of SC 1.1.1 Non-text content. Not sure if this is the best place to share that doubt, so please let me know if I should share it in a different place.

[SC 1.1.1 includes a section with additional information](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html#additional-information). In one of the paragraphs, we can read that for non-text content that is time-based media we need to refer to guideline 1.2 to make it accessible. However, we are reminded that it is important to identify the time-based media element in the page using a text alternative.

The next paragraph deals with live audio-only and live video-only content , and it comes to say that it is much more difficult to  provide text alternatives in this case. We are reminded that "for these types of non-text content, text alternatives provide a descriptive label."

Here's my doubt: what's the reason of having that paragraph on live audio-only and live video-only content? In my opinion, the previous paragraph referring to guideline 1.2 and reminding to identify the media using a text alternative is still valid here.

**Comment by bruce-usab** [comment, marked as answer]
Thank you for your patience @cmuncharaz with this good question!

It is not uncommon for more than one SC to be applicable to a single HTML element.  With Time-Based Media, what an author needs to address SC 1.1.1 is fairly minimal: a text alternative providing descriptive identification.

More robust requirements for Time-based media are found under guideline 1.2.  But even when those are all addressed, the descriptive label required by by SC 1.1.1 is still very important and also required.  The label is not provide the text equivalent, but it is providing essential navigation cues.

> What's the reason of having that paragraph on live audio-only and live video-only content?

That paragraph is providing some explanation as to why the 1.1.1 requirement for Time-Based Media is so modest.
